### PR TITLE
Added prettier

### PR DESCRIPTION
**Pull request by kkruk-sumo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "dist"
   ],
   "scripts": {
-    "build": "node ./scripts/bundle.js && rollup -c"
+    "build": "node ./scripts/bundle.js && rollup -c",
+    "prettier": "prettier --write ."
```
Comment (mgol-sumo) [inline package.json:12]: This should also be enforced in the build process, just not with `--write` but with `--check`. See what we have in Prada for comparison.

Comment (kkruk-sumo) [reply package.json]: Added `husky` pre-commit task.

File: package.json [hunk 2]
```diff
@@ -8,8 +8,17 @@
...
     "build": "node ./scripts/bundle.js && rollup -c",
-    "prettier": "prettier --write ."
+    "prettier-check-all": "prettier --check .",
+    "prettier-format-all": "prettier --write .",
+    "test": "npm run prettier-check-all"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm test",
+      "pre-push": "npm test"
```
Comment (mgol-sumo) [inline package.json:20]: `pre-commit` should be enough, I think. Any reason to do it in `pre-push` as well?

Comment (kkruk-sumo) [reply package.json:20]: Husky uses both in their README https://github.com/typicode/husky#install so I'm not sure whether `pre-push` is needed.

Comment (kkruk-sumo) [reply package.json]: I removed `pre-push` hook.

**Pull request merged**